docs/index.html

```diff
@@ -30,7 +30,7 @@ <h1>AEA Manager</h1>
           <p>Public beta</p>
 
           <a href="#links" class="Link">
-            <button type="button" class="Btn BtnDownload">Download 0.7.5</button>
+            <button type="button" class="Btn BtnDownload">Download 5.4.3</button>
```

Comment:
@VijaySharma815 you should be changing version in the case when we released a new version and the site needs to be updated. "pipenv run update-version **5.4.3**" is a placeholder, should be replaced with real version only when it's released.
This will break the links to a latest builds.
Please revert this change.